spec/PipelineBuilderSpec.php

```diff
@@ -14,11 +14,6 @@ function it_is_initializable()
         $this->shouldHaveType(PipelineBuilder::class);
     }
 
-    function it_should_build_a_pipeline()
-    {
-        $this->build()->shouldHaveType(PipelineInterface::class);
-    }
```

Comment:
Why is this removed?

Reply:
No need for it, the strict type already ensures it without a test.

Reply:
Fair enough 👍